bigtable-client-core-parent/bigtable-hbase/src/test/java/org/apache/hadoop/hbase/client/TestAbstractBigtableConnection.java

```diff
@@ -262,7 +268,53 @@ protected SampledRowKeysAdapter createSampledRowKeysAdapter(
 
     @Override
     public Table getTable(TableName tableName, ExecutorService executorService) {
-      return new AbstractBigtableTable(this, createAdapter(tableName)) {};
+      return new AbstractBigtableTable(this, createAdapter(tableName)) {
```

Comment:
why not call BigtableTableCore(..) here?

Reply:
oops, fixed